cloudhub-2/modules/ROOT/pages/ch2-patch-updates.adoc

```diff
@@ -40,12 +63,12 @@ This upgrade is irreversible. Ensure that you understand the impact of the upgra
 
 . From Anypoint Platform, select *Runtime Manager* > *Private Spaces*.
 +
-If an upgrade is available for a private space, a tooltip shows under the *Infra Status* column.
+If an upgrade is available for a private space, the *Upgrade Available* status appears under the *Infra Status* column.
 . Click *Upgrade available*.
 . Select the checkbox to confirm that you agree to upgrade early.
```

Comment:
In this case, they would have to select 'Upgrade now' radio button and then opt-in for an early upgrade.

Reply:
```suggestion
. Click *Upgrade now* and opt in to the early upgrade.
```